modules/model.py

```diff
@@ -186,13 +186,11 @@ def on_done(feature):
 
             results.append(
                 (
-                    images,
+                    [save_image(img, opts) for img in images],
                     ImageGenerationOptions.parse_obj(
                         {"seed": manual_seed, **opts.dict()}
                     ),
                 )
             )
-            for img in images:
-                save_image(img, opts)
```

Comment:
gr.galleryをPIL.imageではなくimage pathで指定する用に変更
(I/Oが冗長に感じるけどPIL.imageだとpng infoが欠落するので仕方ない)

Changed gr.gallery to specify image path instead of PIL.image.
(I/O seems redundant, but PIL.image lacks png info, so it can't be helped)